apps/designer/app/designer/features/sidebar-left/panels/pages/settings.tsx

```diff
@@ -127,3 +164,149 @@ export const NewPageSettings = ({
     </>
   );
 };
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type EditPageData = FetcherData<{}>;
```

Comment:
I am sure there is a better way to express this type afaik linter suggests different ways when you try to use {}

Reply:
None of the alternative do what I need, and this works perfectly. I think this is an edge case where linter is wrong